Makefile

```diff
@@ -76,9 +57,6 @@ applogs:
 weblogs:
 	docker logs --details -ft $$(docker ps -qf name=$(FRONTEND_CONTAINER_NAME))
 
```

Comment:
I notice we're using two different methods for getting container logs (docker-compose logs and docker logs). Assuming they produce the same output, can we pick one?